server/tidb_test.go

```diff
@@ -43,7 +44,9 @@ var suite = new(TidbTestSuite)
 var _ = Suite(suite)
 
 func (ts *TidbTestSuite) SetUpSuite(c *C) {
-	store, err := tidb.NewStore("memory:///tmp/tidb")
+	store, err := tikv.NewMockTikvStore()
+	tidb.SetStatsLease(0)
+	tidb.SetSchemaLease(0)
```

Comment:
I think this code can be removed. Because we use mock `syncer`.

Reply:
Great!